docs/user-documentation/media.md

```diff
@@ -1,60 +1,79 @@
 Drupal 8 recognizes files (such as images, audio files, video files, etc.) but wraps each file in an intermediate structure called a
-"media" to allow us to attach fields to files. It is in a media's fields that we store information about the media's file, such as file
+"media" to allow us to attach fields to files. Drupal uses a media's fields to store information about the media's file, such as file
 size, width and height (for images), alt text (for images), creation date, and so on.
 
 !!! note "Compared to Islandora 7"
     In Islandora 7, this sort of technical metadata would have been stored in a single RELS-INT datastream.
-    In Islandora 8, each datastream holds its own technical metadata using media entities.
+    In Islandora 8, each datastream holds its technical metadata using an associated media entity.
+    
+Fedora will store the media's file as a binary resource and use the media's properties as the binary resource's description.
```

Comment:
In the fedora context, is getting the media's properties as the binary resource's description true only for media with media use "original file"? For example, for service files I get something like http://127.0.0.1:8080/fcrepo/rest/70/a4/af/75/70a4af75-16cf-4f7e-a88e-2eb744bbe152/fcr:metadata.

Reply:
Yes, this is still true. The reason you get a UUID-based URI for derivatives is that they are being described in Fedora as externally-reference content whereas original files are actually stored in Fedora. In both cases the `/fcr:metadata` for a binary resource is populated by the File's associated Media entity.